Documentation/Testing/Tutorial/Acceptance.rst

```diff
@@ -0,0 +1,267 @@
+    ⏺ Recording ⏺ step-by-step screenshots will be saved to /var/www/html/Tests/../var/log/_output/
+    Directory Format: record_5be441bbdc8ed_{filename}_{testname} ----
+    FrontendPagesCest: First page is rendered
+    Signature: Bk2k\SiteIntroduction\Tests\Acceptance\Frontend\FrontendPagesCest:firstPageIsRendered
+    Test: Acceptance/Frontend/FrontendPagesCest.php:firstPageIsRendered
+    Scenario --
+     I am on page "/"
+      [GET] https://introduction.ddev.site/
+     I see "Open source, enterprise CMS delivering  content-rich digital experiences on any channel,  any device, in any language"
+     I click "Customize"
+     I see "Incredible flexible"
+     PASSED
+
+    ---------------------------------------------------------------------------------------------------------------------------------------------------------------------
+    ⏺ Records saved into: file:///var/www/html/Tests/../var/log/_output/records.html
+
+
+    Time: 8.46 seconds, Memory: 8.00MB
+
+    OK (1 test, 2 assertions)
+
+    lolli@apoc /var/www/local/site-introduction $
+
+Done: Local test execution of a projects acceptance test!
+
+.. index:: Testing; GitHub Actions
+
+GitHub Actions
+==============
+
+With local testing in place, we now want tests to run automatically when something is merged
+into the repository, and when people create pull requests for our project, we want to make sure
+that our carefully crafted test setup actually works. We're going to use Github's Actions CI
+service to get that done. It's free for open source projects.
+To tell the CI what to do, create a new workflow file in
+`.github/workflows/tests.yml <https://github.com/TYPO3-Documentation/site-introduction/blob/master/.github/workflows/tests.yml>`__
+
+.. code-block:: yaml
+
+   name: tests
+
+   on:
+     push:
+     pull_request:
+     workflow_dispatch:
+
+   jobs:
+     testsuite:
+       name: all tests
+       runs-on: ubuntu-20.04
+       steps:
+         - name: Checkout
+           uses: actions/checkout@v2
+
+         - name: Start DDEV
+           uses: jonaseberle/github-action-setup-ddev@v1
+
+         - name: Import database
+           run: ddev import-db --src=./data/db.sql
+
+         - name: Import files
+           run: ddev import-files --src=./assets
+
+         - name: Install Composer packages
+           run: ddev composer install
+
+         - name: Allow public access of var folder
+           run: sudo chmod 0777 ./var
+
+         - name: Run acceptance tests
+           run: ddev exec bin/codecept run acceptance -d -c Tests/codeception.yml
+
+It's possible to see executed test runs `online <https://github.com/TYPO3-Documentation/site-introduction/actions>`_.
+Green :)
+
+
+Summary
+=======
+
+This chapter is a show case how project testing can be done. Our example projects makes it easy for us since
```

Comment:
Below you write "instance" so IMO "project" here
```suggestion
This chapter is a show case how project testing can be done. Our example project makes it easy for us since
```